pkg/webhook/admission/defaulter.go

```diff
@@ -33,9 +33,10 @@ type Defaulter interface {
 }
 
 // DefaultingWebhookFor creates a new Webhook for Defaulting the provided type.
-func DefaultingWebhookFor(defaulter Defaulter) *Webhook {
+func DefaultingWebhookFor(defaulter Defaulter, recoverPanic bool) *Webhook {
```

Comment:
I have a little concern about these public API change, not sure if there were some ppl directly use them.

So how about a new `func (wh *Webhook) WithRecoverPanic*(enable bool) *Webhook` and you can use it in builder/webhook like this `return admission.ValidatingWebhookFor(validator).WithRecoverPanic(blder. recoverPanic)`